formcentral/test.yaml

```diff
@@ -0,0 +1,30 @@
+version: '3'
+
+services:
+  main:
+    depends_on:
+      - postgres
+    build: .
+    image: formcentral
+    environment:
+      POSTGRES_DB: test-db
+      POSTGRES_HOST: postgres
+      POSTGRES_PORT: 5432
+      POSTGRES_USER: test-user
+      POSTGRES_PASSWORD: test-pass
+    command: |
+      bash -c '
+        tools/wait-for-it.sh postgres:5432 -t 60 &&
+        go test
```

Comment:
Let's make sure that the developer can have a loop-on-fail process so that we encourage TDD. 

Currently, I use "looper" locally, which provides a nice experience. 

I don't see an advantage of having the same container with build/test tools launched in production, but I do see an advantage with creating a local dev/build/test container and using that.

Reply:
> Let's make sure that the developer can have a loop-on-fail process so that we encourage TDD.

I'd like to propose we take it a bit further and do loop-on-save. That way devs have an interactive local environment that reloads as-you-work, even if they break or fix issues.

> Currently, I use "looper" locally, which provides a nice experience.

Looks good and works!

Take a look at, https://github.com/vlab-research/fly/pull/23/commits/1b2614d5ed42e135154e90da541a3682b7ae580f.

> I don't see an advantage of having the same container with build/test tools launched in production, but I do see an advantage with creating a local dev/build/test container and using that.

Agreed.